commons/passive/dreem/dreem_hypnogram.avsc

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "record",
+  "namespace": "org.radarcns.passive.dreem",
+  "name": "Dreem",
+  "doc": "AVRO Schema for DREEM hypnogram data",
+  "fields": [
+    {"name": "studyId", "type":"string", "doc":"This is study number"},
+    {"name": "subjId", "type":"string", "doc":"Subject Identifier for the study. Format: xxx-yyy where xxx represents 3 digits for the site ID and yyy represents 3 digits for the participant ID"},
```

Comment:
Study ID and subject ID should both be part of the observation key and be mapped to a managementportal participant. If this information is important to receive, please send it in a separate topic.